assets/js/dispensing.js

```diff
@@ -759,7 +766,10 @@ function gotoAppointmentEncounterType() {
     document.location = "/views/patient/appointment.html?patient_id=" + sessionStorage.patientID;
   } else if (parseInt(sessionStorage.programID) === 2) {
     nextEncounter(sessionStorage.patientID, sessionStorage.programID, true)
-  } else{
+  } 
+  else if(sessionStorage.userRoles == 'Pharmacist' && parseInt(sessionStorage.programID) == 14)
+    window.location = "/";
+    else{
```

Comment:
User roles are saved as an array of roles, please search through it by using the match function in javascript

Reply:
noted we will do that